app.rb

```diff
@@ -1,8 +1,8 @@
 require 'io/console'
-require './modules/book_handle.rb'
-require './modules/people_handle.rb'
-require './modules/rental_handle.rb'
-require './modules/reading_data.rb'
+require_relative './modules/book_handle.rb'
+require_relative './modules/people_handle.rb'
+require_relative './modules/rental_handle.rb'
+require_relative './modules/reading_data.rb'
 
 class App
   attr_reader :books, :person
```

Comment:
### Your Application Start
![Screenshot from 2022-09-07 04-00-42 (copy)](https://user-images.githubusercontent.com/93039359/188755189-acf82ae0-83c1-4a76-b990-45572ccd2e43.png)

**[Optional]**
- [ ] Please remove the things shown in the red box above. Because they are making your UI not look clean :thinking: